app/models/vmdb_database_setting.rb

```diff
@@ -4,7 +4,7 @@ class << self
   end
   self.aar_columns = []
 
-  self.table_name = 'pg_settings'
+  self.table_name = 'pg_catalog.pg_settings'
```

Comment:
It's probably ambiguous in some contexts.  It's probably better to be explicit but I'd be curious how this failed for @matthewd or he just knew to change this.